qemu/tests/qemu_img_lock_reject_boot.py

```diff
@@ -62,8 +61,9 @@ def _verify_write_lock_err_msg(test, output, img_file=None):
     try:
         vm2.create(params=vm2.params)
         vm2.verify_alive()
-    except VMCreateError as err:
-        _verify_write_lock_err_msg(test, err.output, img_file)
+    except:
+        output = vm2.process.get_output().strip()
+        _verify_write_lock_err_msg(test, output, img_file)
```

Comment:
Catch all exceptions is accepted, but i wonder why the exception message comes from vm.process? does it cover all kinds of VM creation failed?

Reply:
Yep, there is a corner case of `MonitorConnectError` when the VM is actually already dead, but it also means we can't get the expected error message as it is outside of context of `MonitorConnectError` excetion raised.